plugins/collaajUpdatePlugin/lib/api/KalturaCaptureSpaceUpdateResponse.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class KalturaCaptureSpaceUpdateResponse extends KalturaObject {

```

Comment:
This was not done by me, but rather Gonen.
I don’t know to which end.

Thanks,
Kobi M.

From: Tan-Tan [mailto:notifications@github.com]
Sent: Tuesday, November 10, 2015 15:39 PM
To: kaltura/server <server@noreply.github.com>
Cc: Kobi Michaeli <kobi.michaeli@kaltura.com>
Subject: Re: [server] Kajam 11.2.0 plat 3434 (#3465)


In plugins/collaajUpdatePlugin/lib/api/KalturaCaptureSpaceUpdateResponse.php<https://github.com/kaltura/server/pull/3465#discussion_r44406335>:

> @@ -0,0 +1,18 @@

> +<?php

> +

> +class KalturaCaptureSpaceUpdateResponse extends KalturaObject {

What's the purpose of this class, it's just wrapping info.
Why won't it contain the content of info instead?

—
Reply to this email directly or view it on GitHub<https://github.com/kaltura/server/pull/3465/files#r44406335>.